### ISSUE TITLE
fillForm is broken

### ISSUE THREAD

**Issue by stocksp** [MeteorPackaging/pdftk#6]
I posted this in the Meteor forums,  as I think it's a Meteor bug ... maybe not?!
After installing the package:
Run meteor and inspect the pdftk_pdftk.js Meteor package system has generated. It lives in
.meteor/local/build/programs/server/packages.
line 70 has a blank space appended to 'fill_form' the second parameter
return PDFTK.execute([pdf, 'fill_form ', xfdf, 'output', output], callback);

If you examine the source of the package on GitHub, you can see the original file is fine and not the source of the bad parameter.
Running PDFTK.execute directly with the proper name (no space) and fillForm works correctly

**Comment by aadamsx** [comment]
What operating system are you on?

**Comment by stocksp** [comment]
Mac Os

**Comment by aadamsx** [comment]
What version?

**Comment by stocksp** [comment]
OS X Yosemite

**Comment by stocksp** [comment]
What is strange is  that the bug in the file was fixed in pull request #3 six months ago.

**Comment by aadamsx** [comment]
Pdftk should work on that version. On OS X 10.11 it really is broke. How did you install pdftk?  Did you use pdf labs install?  If not, I would give that a try. That install should work with OS X 10.10.

**Comment by aadamsx** [comment]
I see that bug you are referring to. Are you actully getting that error?  I have my own pdftk package that should work.

**Comment by stocksp** [comment]
pdftk is working fine.  I can actually run the 'form_fill' from the command line.   It must just be that this repo has just not be pushed to Atmosphere and its using the old code.   I'll probably just clone the repo and fix it locally.   I got it running using  PDFTK.execute, so I'm good.   Hopefully the package owners will get around to updating Atmosphere

**Comment by aadamsx** [comment]
Yeah, don't count on it -- that's why i started my own repo.  By the way, watch out upgrading to 10.11, pdftk on the command line will not run form fills (or anything else for that matter).  I found out the hard way.  Good luck.

**Comment by andreixk** [comment]
They fixed it (sorta) as answered [here](https://stackoverflow.com/questions/32505951/pdftk-server-on-os-x-10-11). You can download the pdftk for OSX 10.11 [here](https://www.pdflabs.com/tools/pdftk-the-pdf-toolkit/pdftk_server-2.02-mac_osx-10.11-setup.pkg) but the app is still buggy - it can't open PDFs that have owner password but no user password (i.e. files it SHOULD be able to open).
